deps/wazuh_testing/wazuh_testing/tools/system.py

```diff
@@ -571,24 +590,38 @@ def remove_package(self, host, system, package_uninstall_name=None, custom_unins
         if custom_uninstall_playbook:
             remove_operation_result = self.run_playbook(host, custom_uninstall_playbook)
         elif package_uninstall_name:
-            if os_name == 'windows':
-                remove_operation_result = self.get_host(host).ansible("win_command",
-                                                                      f"{package_uninstall_name} /uninstall /quiet /S",
-                                                                      check=False)
-            elif os_name == 'linux':
-                os = self.get_host_variables(host)['os'].split('_')[0]
-                if os == 'centos':
-                    remove_operation_result = self.get_host(host).ansible("yum",
-                                                                          f"name={package_uninstall_name} state=absent",
-                                                                          check=False)
-                elif os == 'ubuntu':
-                    remove_operation_result = self.get_host(host).ansible("apt",
-                                                                          f"name={package_uninstall_name} state=absent",
-                                                                          check=False)
-            elif os_name == 'macos':
-                remove_operation_result = self.get_host(host).ansible("command",
-                                                                      f"brew uninstall {package_uninstall_name}",
-                                                                      check=False)
+            if use_npm:
+                if system == 'macos':
+                    cmd = f"PATH=/opt/homebrew/bin:$PATH npm uninstall -g {package_uninstall_name}"
+                    remove_operation_result = self.get_host(host).ansible("shell", cmd, check=False)
+                elif system == 'windows':
+                    cmd = f"npm uninstall -g {package_uninstall_name}"
+                    remove_operation_result = self.get_host(host).ansible("win_shell", cmd, check=False)
+                else:
+                    cmd = f"npm uninstall -g {package_uninstall_name}"
+                    remove_operation_result = self.get_host(host).ansible("shell", cmd, check=False)
+
+                logging.info(f"npm package removed result {remove_operation_result}")
+                return remove_operation_result
```

Comment:
As before, you should remove this return, but this time the return on line 628 is properly tabulated.

Reply:
Fixed here: https://github.com/wazuh/wazuh-qa/pull/5092/commits/b924e9a048a6905ccfb4bac306e5f9f7668bde60